.aws/src/main.ts

```diff
@@ -255,6 +304,8 @@ class ListAPI extends TerraformStack {
               `arn:aws:secretsmanager:${region.name}:${caller.accountId}:secret:${config.name}/${config.environment}/*`,
               `arn:aws:secretsmanager:${region.name}:${caller.accountId}:secret:${config.prefix}`,
               `arn:aws:secretsmanager:${region.name}:${caller.accountId}:secret:${config.prefix}/*`,
+              // Add Dev RDS-specific secrets if in Dev environment
+              ...(config.isDev ? secretResources : []),
             ],
```

Comment:
I _think_ I got this right and these should be conditionally added under `taskExecutionRolePolicyStatements`.